.circleci/config.yml

```diff
@@ -39,6 +34,11 @@ executors:
       - image: redis:5.0.8
 
 commands:
+  install-browser-tools-no-firefox:
```

Comment:
dang, forgot about this option, got too deep into how GitLab uses YAML for configuration

thanks!